packages/kit/src/exports/vite/index.js

```diff
@@ -272,13 +271,6 @@ function kit({ svelte_config }) {
 					__SVELTEKIT_EMBEDDED__: kit.embedded ? 'true' : 'false'
 				};
 
-				// This ensures that esm-env is inlined into the server output with the
```

Comment:
I was reading the updated comment and couldn't remember what the issue was. It'd probably be good to keep this more detailed comment

Reply:
done